projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/agents-button/agents-button.component.scss

```diff
@@ -15,7 +15,7 @@ aca-agents-button.aca-agents-button {
 
       .aca-agents-button-icon {
         display: flex;
-        align-self: baseline;
+        align-self: center;
```

Comment:
Icon seems not aligned actually, could you please align icon to text? Text should be in the middle of icon. 
Actually: 
<img width="209" alt="image" src="https://github.com/user-attachments/assets/b818ce06-4537-42a6-98a2-edacd5810ed3">


In Figma: 
<img width="364" alt="image" src="https://github.com/user-attachments/assets/93515df3-740b-4432-8399-ef732be54236">